runtime/runtime/src/congestion_control.rs

```diff
@@ -388,12 +388,30 @@ impl ReceiptSinkV2 {
     fn try_forward(
         receipt: Receipt,
         gas: u64,
-        size: u64,
+        mut size: u64,
         shard: ShardId,
         outgoing_limit: &mut HashMap<ShardId, OutgoingLimit>,
         outgoing_receipts: &mut Vec<Receipt>,
         apply_state: &ApplyState,
     ) -> Result<ReceiptForwarding, RuntimeError> {
+        // There is a bug which allows to create receipts that are above the size limit. Receipts
+        // above the size limit might not fit under the maximum outgoing size limit. Let's pretend
+        // that all receipts are at most `max_receipt_size` to avoid receipts getting stuck.
+        // See https://github.com/near/nearcore/issues/12606
+        let max_receipt_size = apply_state.config.wasm_config.limit_config.max_receipt_size;
+        if size > max_receipt_size {
+            if size > max_receipt_size {
+                tracing::warn!(
+                    target: "runtime",
+                    "try_forward observed a receipt with size exceeding the size limit! receipt_id: {} size: {} size_limit: {}",
```

Comment:
nit: use structured logging - set the values as fields rather than in the formatted message
```
tracing::warn!(
  receipt_id=?..
  size,
  max_size,
  "try_forward observed a receipt with size exceeding the size limit"
);
```